### PR TITLE
feat(docs): Document how to do l10n extraction/merge.

### LINKED ISSUE #35: Extraction tutorial
@shane-tomlinson Here's a GIF of me extracting strings from `fxa-content-server` and merging the new templates in this repo.

l10n

The last step (not recorded) is opening a PR here and waiting for mathjazz's okay to merge.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
fixes #35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,24 @@ The moz-verbatim translation team works with this L10N repo, pushing changes int
...
+The process to extract strings:
+
+```bash
+> cd <fxa-auth-mailer root>
+> grunt l10n-extract
+> cp server.pot <fxa-content-server root>/locale/templates/LC_MESSAGES/
+> cd <fxa-content-server root>
+> grunt l10n-extract
+> cp -r ./locale/templates/ <fxa-content-server-l10n root>/locales/templates
+> cd <fxa-content-server root>
```
Comment (pdehaan) [inline README.md]: What dis? Aren't we already in `{{fxa-content-server root}}` per line 22?
Should one of those be "*-l10n"?

Comment (shane-tomlinson) [inline README.md]: You are right @pdehaan - I thought "ah, it's a doc only change, I'll just self-merge." You caught me.

Comment (shane-tomlinson) [inline README.md]: I just took that liberty again, updated in master, and pushed - see b3f2a606dddda57676c1e52d8c699d93d893054e

Comment (pdehaan) [inline README.md]: Thanks, sir!

**Pull request merged**